File: php/scripts/cost_center.php
<?php
require_once(__DIR__."/../vendor/autoload.php");
require_once("utils.php");
use Namelivia\TravelPerk\ServiceProvider;
$isSandbox = false;
$travelperk = (new ServiceProvider())->build(getenv("API_KEY"), false);
$costCenter = $travelperk->costCenters()->costCenters()->get("1");
write_output("cost_center", [
	$costCenter->id,
	$costCenter->name,
	$costCenter->archived,
	$costCenter->countUsers,
	count($costCenter->users),
    $costCenter->users[0]->firstName,
    $costCenter->users[0]->lastName,
    $costCenter->users[0]->email,
    $costCenter->users[0]->id,
    $costCenter->users[0]->phone,
    $costCenter->users[0]->profilePicture,
]);

File: python/scripts/cost_center.py
import os
from travelperk_http_python.builder.builder import build
from utils import write_output
travelperk = build(os.environ["API_KEY"], False)
cost_center = travelperk.cost_centers().cost_centers().get("1")
write_output("cost_center", [
    cost_center.id,
    cost_center.name,
    cost_center.archived,
    cost_center.count_users,
    cost_center.users,
    cost_center.users[0].first_name,
    cost_center.users[0].last_name,
    cost_center.users[0].email,
    cost_center.users[0].id,
    cost_center.users[0].phone,
    cost_center.users[0].profile_picture,
])
